lib/nucleus_core/worker.rb

```diff
@@ -0,0 +1,48 @@
+module NucleusCore
+  class Worker
+    class Adapter
+      def self.call(class_name, method_name, args={})
+        execute(class_name, method_name, args)
+      end
+
+      def call(class_name, method_name, args={})
+        self.class.execute(class_name, method_name, args)
+      end
+
+      def self.execute(class_name, method_name, args={})
```

Comment:
execute is the method the framework adapter needs to define, create new method to do `Object.const_get(class_name).new(args).send(method_name)` 🤔